jobs/views.py

```diff
@@ -276,6 +376,12 @@ def get_form_kwargs(self):
             kwargs['initial'] = {'email': self.request.user.email}
         return kwargs
 
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data()
+        ctx.update(kwargs)
+        ctx['needs_preview'] = not self.request.user.is_staff
```

Comment:
1. I will switch to that role.
2. I can change that, in which case I'd introduce specific new views for the review workflow. The main issue is that while editing I want to enforce preview for normal users. For reviewers however that's not needed, as they will be redirected back to the review screen anyway.